README.md

```diff
@@ -57,9 +57,19 @@ class Foo
 You can then pick one of the [implementations](implementation-url) of the interface to get a clock.
 
 If you want to implement the interface, you can require this package and
-implement `Psr\Clock\ClockInterface` in your code. Please read the
-[specification text](specification-url)
-for details.
+implement `Psr\Clock\ClockInterface` in your code. 
+
+Don't forget to add `psr/clock-implementation` to your `composer.json`s `provides`-section like this:
```

Comment:
This is already done in the `Installation` step, should we repeat this here?

Reply:
The Interface Implementors need to add that part to their implememtations `composer. json` so that their package can be found in the installation step.

Yes, one could argue that the first mention should be enough. But to be explicit about that eases work for implementors while it doesn't harm the users

Reply:
Fair enough, thanks.